### PR TITLE
feat: add Connector docs to LB4 docs

### PR DESCRIPTION
**Pull request by hacksparrow**
Add Connector docs to LB4 docs.

Required to fix https://github.com/strongloop/loopback-next/issues/2598.

### CODE REVIEW

File: docs/site/sidebars/lb4_sidebar.yml [hunk 1]
```diff
@@ -415,6 +414,194 @@ children:
     url: Download-examples.html
     output: 'web, pdf'
 
+- title: 'Connectors reference'
+  url: Connectors-reference.html
```
Comment (raymondfeng) [inline docs/site/sidebars/lb4_sidebar.yml]: https://loopback.io/doc/en/lb4/Connectors-reference.html does not exist. Am I missing something here?

Comment (raymondfeng) [reply docs/site/sidebars/lb4_sidebar.yml]: Just realized https://github.com/strongloop/loopback.io/pull/863

Comment (hacksparrow) [reply docs/site/sidebars/lb4_sidebar.yml]: There are many more "external" files, all of which are present in the `lb3` dir or made available there eventually. Those files are copied over to the `lb4` dir by the build process in `loopback.io`.

File: docs/site/sidebars/lb4_sidebar.yml [hunk 2]
```diff
@@ -415,6 +414,194 @@ children:
...
+
+    - title: 'Cassandra connector'
+      url: Cassandra-connector.html
+      output: 'web, pdf'
+
+    - title: 'Cloudant connector'
+      url: Cloudant-connector.html
+      output: 'web, pdf'
+
+    - title: DashDB
```
Comment (jannyHou) [inline docs/site/sidebars/lb4_sidebar.yml]: `DashDB` --> `DashDB connector` to be consistent with others?

File: docs/site/sidebars/lb4_sidebar.yml [hunk 3]
```diff
@@ -415,6 +414,194 @@ children:
...
+  - title: 'Database connectors'
+    url: Database-connectors.html
+    output: 'web'
+    children:
+
+    - title: 'Cassandra connector'
+      url: Cassandra-connector.html
+      output: 'web, pdf'
+
+    - title: 'Cloudant connector'
```
Comment (jannyHou) [inline docs/site/sidebars/lb4_sidebar.yml]: nitpick: same as https://github.com/strongloop/loopback.io/pull/863#discussion_r309302677, we also support couchdb2. If it takes time to add the new entry we can do it in a separate PR.

Comment (hacksparrow) [reply docs/site/sidebars/lb4_sidebar.yml]: Let's do it in a separate PR.
